### DISCUSSION TITLE
Release 23.01

### DISCUSSION THREAD

**Discussion by svpcom** [svpcom/wfb-ng#271] [General]
## Changes:
1. FEC settings now embedded into session packet. This means that RX get FEC type, K and N from TX. This allows to add dynamic FEC in future without protocol change and allows different FEC settings for uplink and downlink (this settings now in TX sections only)
2. Small tunneled IP packets now aggregated the same way as mavlink packets. This raise performance for interactive TCP sessions via tunnel.

## Changes:

1. RTSP server now enabled by default. So you can see video simultaneously on RPI (via HDMI) and on laptop via QGC
2. In rtl8812au driver was added a patch that drop tx packets while medium is busy and internal hardware queue overflows. This allows to reduce latency by avoid sending stale data and increase FEC efficiency.

- This image was tested on **RPI3b + Alfa AWUS036ACH + PI Camera**. Use native **("thick") USB3 cable** from this card. If you have issues with different HW setup then try to test this setup first before issuing bug report. 
- **OSD** will not work on **RPI4** due to dropped **OpenVG** support. 
- If you use high-power cards like "taobao card" then you **must** use external high-current power supply and [low-esr capacitor](https://github.com/svpcom/wifibroadcast/wiki/Setup-HOWTO) on the card. **Don't power them from USB!**

<hr /><em>This discussion was created from the release <a href='https://github.com/svpcom/wfb-ng/releases/tag/wfb-ng-23.01'>Release 23.01</a>.</em>

**Comment by DangerD1024** [comment]
Will it work RPI 4B?
Tried to start it, but there's no wifi driver...

**Reply by svpcom** [reply]
```cd rtl8812au && make && sudo make install```
